src/modules/swap/ModuleSwap.vue

```diff
@@ -901,8 +903,10 @@ export default {
       this.addressValue = {};
       this.selectedProvider = {};
       this.localGasPrice = '0';
-      this.$refs.toToken.clear();
-      this.$refs.amountInput.clear();
+      if (this.mounted) {
```

Comment:
no need for the mounted variable. just check for whether they're not null when calling `this.clear`. before calling them.